app/views/organizations/staff/pets/_pet_cards.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-lg-4 col-md-6 p-0">
+  <%= link_to adoptable_pet_path(pet), class: 'text-decoration-none' do %>
+    <div class="card card-hover m-4" style='overflow: hidden'>
+      <% image = image_tag('coming_soon.jpg', class: 'card-img-top pet-card') %>
+      <% image = image_tag pet.images.first, class: 'card-img-top pet-card' if pet.images.attached? %>
+
+      <%= image %>
+      <ul class="list-group list-group-flush">
+        <li class="list-group-item d-flex justify-content-between">
+          <h5 class="card-title text-secondary mb-0">
+            <%= pet.name %>
+          </h5>
+        </li>
+        <% li_classes = %w[list-group-item text-secondary] %>
+        <%= tag.li("Sex: #{pet.sex}", class: li_classes) %>
```

Comment:
@kasugaijin Do we need the breed in this card? The issue lists it but the current table does not have breed.

Reply:
@jmilljr24 @sarvaiyanidhi we removed breed from the table in a PR today. So can remove from the cards also :)